x-pack/plugins/cases/server/client/cases/get.ts

```diff
@@ -232,6 +234,86 @@ export const get = async (
   }
 };
 
+/**
+ * Retrieves a case resolving its ID and optionally loading its comments and sub case comments.
+ *
+ * @experimental
+ */
+export const resolve = async (
+  { id, includeComments, includeSubCaseComments }: GetParams,
+  clientArgs: CasesClientArgs
+): Promise<CaseResolveResponse> => {
+  const { unsecuredSavedObjectsClient, caseService, logger, authorization } = clientArgs;
+
+  try {
+    if (!ENABLE_CASE_CONNECTOR && includeSubCaseComments) {
+      throw Boom.badRequest(
+        'The `includeSubCaseComments` is not supported when the case connector feature is disabled'
+      );
+    }
+
+    const {
+      saved_object: savedObject,
+      ...resolveData
+    }: SavedObjectsResolveResponse<CaseAttributes> = await caseService.getResolveCase({
+      unsecuredSavedObjectsClient,
+      id,
+    });
+
+    await authorization.ensureAuthorized({
+      operation: Operations.getCase,
```

Comment:
I think we should use another operation for resolving a case. @jonathan-buttner What do you think?

Reply:
I think it's probably fine to use `getCase` here because the operation is essentially the same. If a user has the privileges to resolve a case, they should be able to get a case too right?

Reply:
You are right about that. My concern is mostly about logging and traceability. How we will distinguish the two calls from the logs?

Reply:
@cnasikas @jonathan-buttner You both are right, it is the same operation but I did not think about the logs, I can create another operation so we can trace it properly.

Reply:
Never mind, Christos was right haha 😆